activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -130,8 +130,11 @@ def test_schema_names_logs_name
       def test_statement_key_is_logged
         bind = Relation::QueryAttribute.new(nil, 1, Type::Value.new)
         @connection.exec_query("SELECT $1::integer", "SQL", [bind], prepare: true)
```

Comment:
@byroot You could duplicate this test and set `prepare: false` (or leave out of args) to replicate the issue.

Reply:
No, it's not as simple as that unfortunately.